tests/rsync/rsync_spec.lua

```diff
@@ -181,6 +182,60 @@ describe("rsync", function()
                 end)
             end)
         end)
+
+        describe("hooks", function()
+            it("success, on_exit called, on_error not called", function()
+                setup(function()
+                    Res_code = -1
+                    Res_data = ""
```

Comment:
```suggestion
                    res_data = ""
```

Use snake case

Reply:
This is snake case, but the linter complains again, cause this is a global variable, global variables start with upper case.

Reply:
Use `local res_data`

Reply:
Done